fulfil_client/serialization.py

```diff
@@ -147,3 +170,24 @@ def default(self, obj):
         '__class__': 'Decimal',
         'decimal': str(o),
     })
+JSONEncoder.register(
+    exceptions.ClientError,
```

Comment:
Why we have to encode this? What am I missing here?

Reply:
This weirdness is for celery to propagate exceptions, but I don't think this will work either because of how celery works.

Reply:
I doubt if this will work. I think this should be handled in task wrapper.

Reply:
OK